pruning/keep_last_n_test.go

```diff
@@ -90,7 +90,7 @@ func TestKeepLastN(t *testing.T) {
 				stubSnap{"a2", false, o(12)},
 			},
 			rules: []KeepRule{
-				MustKeepLastN(3, "a"),
+				MustKeepLastN(4, "a"),
```

Comment:
Why this change?

Reply:
It tests `keep_last_n` works correctly in situation, where number of snapshots `< n`. See https://github.com/zrepl/zrepl/issues/691 for details.